tsconfig.tests.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
+    "declaration": true,
+    "composite": true,
+    "outDir": "./tests/dist",
+    "noEmit": false
```

Comment:
Shouldn't this be `"noEmit": true`?

Reply:
Using project references means that you do need to generate declaration files. This acts as a kind of cache for TS, making things run a bit faster.

These files are outputted to `./tests/dist`, as specified in `outDir`